### PR TITLE
Implement cached_network_image

### PR DESCRIPTION
**Pull request by kubaolszewski**

### CODE REVIEW

File: lib/app/custom_widgets/event_text_sample.dart [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 
   @override
   Widget build(BuildContext context) {
     return Row(
       mainAxisAlignment: MainAxisAlignment.start,
       children: [
-        Text(time, style: CommonTextStyles.basicWhiteText),
-        const SizedBox(width: 5),
-        Text(player, style: CommonTextStyles.basicWhiteText),
+        if (isHomeTeam == true) ...[
```
Comment (MBanaszynski) [inline lib/app/custom_widgets/event_text_sample.dart:16]: What is the difference between the states?
Looks like same widgets

File: lib/repositories/home_screen_repository.dart [hunk 2]
```diff
@@ -7,7 +7,12 @@ class HomeScreenRepository {
   final LiveMatchesRemoteService liveMatchesRemoteService;
 
   Future<List<LiveMatchModel>?> fetchMatchesByDate(String date) async {
-    // return (await liveMatchesRemoteService.fetchMatchesByDate(league: '39', season: '2022', date: date)).response;
+    // return (await liveMatchesRemoteService.fetchMatchesByDate(season: '2023', date: date))
```
Comment (MBanaszynski) [inline lib/repositories/home_screen_repository.dart:10]: Remove commented code

File: lib/app/features/home/home_view.dart [hunk 3]
```diff
@@ -22,9 +23,20 @@ class HomeView extends StatelessWidget {
...
       appBar: CustomAppBar(
-        title: const Text(
-          'scorelive',
-          style: TextStyle(color: Colors.white, fontWeight: FontWeight.bold),
+        title: const SizedBox(
+          width: 125,
+          child: Stack(
+            children: [
+              Text(
+                'scorelive',
```
Comment (MBanaszynski) [inline lib/app/features/home/home_view.dart]: Add to localise

File: lib/app/custom_widgets/live_match_tile/elements/live_match_tile_logo.dart [hunk 4]
```diff
@@ -12,7 +13,7 @@ class LiveMatchTileLogo extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     const String stringPlaceholder = '';
-    final String assetName = liveMatch.league?.flag ??
+    final String flag = liveMatch.league?.flag ??
```
Comment (MBanaszynski) [inline lib/app/custom_widgets/live_match_tile/elements/live_match_tile_logo.dart:16]: Consider using static values for default urls etc, like in `AppColors`

**Review by MBanaszynski — CHANGES_REQUESTED** [review]

File: lib/app/custom_widgets/event_text_sample.dart (hunk 1 above)
Comment (kubaolszewski) [reply lib/app/custom_widgets/event_text_sample.dart:16]: time of an event is displayed on the right or left side of player depending on if its home team or away team

File: lib/repositories/home_screen_repository.dart (hunk 2 above)
Comment (kubaolszewski) [reply lib/repositories/home_screen_repository.dart:10]: left it in order to not overuse api calls

File: lib/app/custom_widgets/live_match_tile/elements/live_match_tile_logo.dart (hunk 4 above)
Comment (kubaolszewski) [reply lib/app/custom_widgets/live_match_tile/elements/live_match_tile_logo.dart:16]: going to do that in separate file exactly like AppColors

**Review by MBanaszynski — APPROVED** [review]

**Pull request merged**